symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/ActivityRegistry.java

```diff
@@ -28,9 +31,37 @@
   /** The bot session forwarded to command-based activities only */
   private final UserV2 botSession;
 
+  private final MessageService messageService;
+
+  private SlashCommand helpCommand;
+
   /** The Datafeed real-time events source, or Datafeed listener */
   private final Consumer<RealTimeEventListener> realTimeEventsSource;
 
+  public ActivityRegistry(UserV2 botSession, Consumer<RealTimeEventListener> realTimeEventsSource , MessageService messageService) {
+    this.botSession = botSession;
+    this.realTimeEventsSource = realTimeEventsSource;
+    this.messageService = messageService;
+
+    this.registerHelpCommand();
+  }
+
+  private void registerHelpCommand() {
+    this.helpCommand = SlashCommand.slash("/help", commandContext -> {
+      List<String> commands = new ArrayList<>();
+      for (AbstractActivity<?, ?> activity : activityList) {
+        ActivityInfo info = activity.getInfo();
+        if (info instanceof CommandActivityInfo) {
+          CommandActivityInfo commandInfo = (CommandActivityInfo) info;
+          commands.add("<li>" + commandInfo.commandName() + ": " + commandInfo.summary() + "</li>");
+        }
+      }
+      String message = "<ul>" + String.join("\n", commands) + "</ul>";
```

Comment:
Use a template if possible

Reply:
The only issue I see is that we don't know at this level what will be the selected template engine

Reply:
maybe just using String.format and building the message in order (that is add the <ul> first then the lines, then </ul>) would make it more template like?